### DISCUSSION TITLE
Unable to move a window other than side tab bar

### DISCUSSION THREAD

**Discussion by jlc488** [Floorp-Projects/Floorp#782] [Feature Request]
![image](https://github.com/Floorp-Projects/Floorp/assets/1614484/e5679713-56e7-4ddf-ba8f-c8f8a835bb58)

When I need to move windows I have to click on the left side tab bar only. 
The problem is if I have so many tabs opened with full length. It is really hard to find the empty space to click and move. 
So I am suggesting Is it possible to make other part of the browser to click and move.

How about a right side bar? or the toolbar ?

Thanks.

**Comment by breadbeforerice** [comment]
You can use toolbar customization to create free space for movement
https://support.mozilla.org/en-US/kb/customize-firefox-controls-buttons-and-toolbars
